cpp/lib/ome/xml/meta/Convert.cpp

```diff
@@ -92,7 +92,6 @@ namespace
       convertFileAnnotations();
       convertListAnnotations();
       convertLongAnnotations();
-      convertMapAnnotations();
```

Comment:
This will be tricky for a bit. C++ support is going to be one schema version behind so that we can get the work into a stable state ASAP.